### ISSUE TITLE
Requiring unknown module @ali/react-native-lazyloadview

### ISSUE THREAD

**Issue by waleedarshad-vf** [IskenHuang/react-native-scrollview-lazyload#3]
Got this error on adding this line `var LazyloadView = require('@ali/react-native-lazyloadview');`

**Comment by roysG** [comment]
Me too, any news?

**Comment by waleedarshad-vf** [comment]
Not Yet! @author should pay attention to it. Please mention your discovery here if you find any resource work for you. i need to implement lazyload also into my app. Thanks

**Comment by waleedarshad-vf** [comment]
@roysG anything work for you?

**Comment by waleedarshad-vf** [comment]
@IskenHuang Need your help. can you resolve it?

**Comment by roysG** [comment]
I stopped work with this plugin cause i found that ListView have already auto lazyload.
@waleedarshad-vf

**Comment by waleedarshad-vf** [comment]
yes you are right but as far as it is concerned i found that if i implement that library https://github.com/magicismight/react-native-lazyload images seems to me a little bit load faster when it appear in viewport as compared to default behavior of listview lazyload images but got stuck with this issue  https://github.com/magicismight/react-native-lazyload/issues/17
